sbi/inference/base.py

```diff
@@ -565,6 +568,16 @@ def __setstate__(self, state_dict: Dict):
         self.__dict__ = state_dict
 
 
+# Refactoring following #1175. As commented in
+# `sbi.utils.user_input_checks.wrap_as_joblib_efficient_simulator`
+# this implementation relies on a double cast torch -> numpy -> torch that
+# allows joblib to iterate over numpy arrays instead of torch tensors. This
+# allows for a final speedup of rougly 10x. Getting rid of the casts adds an
+# additional 3x speedup (at least on my machine). Unfortunately, due to the
+# casts are necessary in order to not introduce breaking changes in project
+# that are currently depending on SBI, but a general restructuring of the
+# simulation pipeline should be considered in the future, leaving the
+# opportunity to the user to simulate natively in numpy.
```

Comment:
👍 
For merging into `main` I would suggest reducing the amount of text here and moving it to the PR description and to an issue describing the suggested general changes to the `simultor` processing pipeline.